### DISCUSSION TITLE
[Slot] Using <Slot /> with react-native

### DISCUSSION THREAD

**Discussion by erickreutz** [radix-ui/primitives#1887] [Ideas] [answered]
I'd love to be able to use `Slot` with react native and at first glance, the only thing that seems to be preventing it is typescript errors. It's a valuable paradigm for both native and web. Do you have any plans to support it?

**Comment by andy-hook** [comment, marked as answer]
Hey @erickreutz , it's great to hear you're finding this pattern useful, though we don't have any plans to support native at this time.

I've converted to a discussion in case others want to show their support for this in the future.

Hope this helps

**Comment by joeyfigaro** [comment]
Throwing in my support for this.

**Comment by Davidcharlie381** [comment]
I actually do advocate for radix supporting react native not only on web, but on mobile as well.

**Comment by codinsonn** [comment]
For those still searching for something like Radix for react-native, you can check out [rn-primitives](https://rnprimitives.com/) by @roninoss

**Reply by Darex1991** [reply]
Really thanks @codinsonn :)
